### PR TITLE
[zh] Sync contribution guidelines (2)

### PR DESCRIPTION
**Pull request by tengqm**
This PR is about the changes to reference generation.

### CODE REVIEW

File: content/zh/docs/contribute/generate-ref-docs/kubernetes-components.md [hunk 1]
```diff
@@ -1,413 +1,49 @@
...
 content_type: task
----
-
-<!--
+weight: 120
 ---
 title: Generating Reference Pages for Kubernetes Components and Tools
 content_type: task
----
+weight: 120
```
Comment (SataQiu) [inline content/zh/docs/contribute/generate-ref-docs/kubernetes-components.md:8]: 此处预览，格式错乱，修修正，https://deploy-preview-22242--kubernetes-io-master-staging.netlify.app/zh/docs/contribute/generate-ref-docs/kubernetes-components/

File: content/zh/docs/contribute/generate-ref-docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+`github.com/website.` The remaining steps refer to your base directory as
+`<web-base>`.
+
+{{< note>}}
+If you want to change the content of the component tools and API reference,
+see the [contributing upstream guide](/docs/contribute/generate-ref-docs/contribute-upstream).
+{{< /note >}}
+-->
+确定你的克隆副本的根目录。例如，如果你按照前面的步骤获取了仓库，你的根目录
+会是 `github.com/website.`。接下来的步骤中，`<web-base>` 用来指代你的根目录。
```
Comment (SataQiu) [inline content/zh/docs/contribute/generate-ref-docs/quickstart.md:56]:
```suggestion
会是 `github.com/website`。接下来的步骤中，`<web-base>` 用来指代你的根目录。
```

File: content/zh/docs/contribute/generate-ref-docs/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+   repository under locations specified in the configuration file.
+1. Updates `kubectl` command links from `kubectl`.md to the refer to
+   the sections in the `kubectl` command reference.
+-->
+脚本 `update-imported-docs` 执行以下步骤：
+
+1. 克隆配置文件中所指定的相关仓库。就生成参考文档这一目的而言，要克隆的
+   仓库默认为 `kubernetes-sigs/reference-docs`。
+1. 在所克隆的仓库下运行命令，准备文档生成器，之后生成 HTML 和 Markdown 文件。
+1. 将所生成的 HTML 和 Markdown 文件复制到 `<web-base>` 本地客隆副本中，
```
Comment (SataQiu) [inline content/zh/docs/contribute/generate-ref-docs/quickstart.md:144]:
```suggestion
1. 将所生成的 HTML 和 Markdown 文件复制到 `<web-base>` 本地克隆副本中，
```

File: content/zh/docs/contribute/generate-ref-docs/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+   放在配置文件中所指定的目录下。
+1. 更新 `kubectl.md` 文件中对 `kubectl` 命令文档的连接，使之指向 `kubectl`
```
Comment (SataQiu) [inline content/zh/docs/contribute/generate-ref-docs/quickstart.md:146]:
```suggestion
1. 更新 `kubectl.md` 文件中对 `kubectl` 命令文档链接，使之指向 `kubectl`
```

File: content/zh/docs/contribute/generate-ref-docs/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+   命令参考中对应的节区。
+
+<!--
+When the generated files are in your local clone of the `<web-base>`
+repository, you can submit them in a [pull request](/docs/contribute/start/)
+to `<web-base>`.
+-->
+当所生成的文件已经被放到 `<web-base>` 目录下，你就可以将其提交给你的派生副本中，
```
Comment (SataQiu) [inline content/zh/docs/contribute/generate-ref-docs/quickstart.md:154]:
```suggestion
当所生成的文件已经被放到 `<web-base>` 目录下，你就可以将其提交到你的派生副本中，
```

File: content/zh/docs/contribute/generate-ref-docs/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,405 @@
...
+
+The `release.yml` configuration file contains instructions to fix relative links.
+To fix relative links within your imported files, set the`gen-absolute-links`
+property to `true`. You can find an example of this in
+[`release.yml`](https://github.com/kubernetes/website/blob/master/update-imported-docs/release.yml).
+-->
+## 修复链接
+
+配置文件 `release.yml` 中包含用来修复相对链接的指令。
+若要修复导入文件中的相对链接，将 `gen-absolute-links` 属性为 `true`。
```
Comment (SataQiu) [inline content/zh/docs/contribute/generate-ref-docs/quickstart.md:292]:
```suggestion
若要修复导入文件中的相对链接，将 `gen-absolute-links` 属性设置为 `true`。
```

**Comment by tengqm** [comment]
@SataQiu All addressed. Thanks for the review.

**Pull request merged**